synfig-studio/src/gui/workarea.cpp

```diff
@@ -391,28 +390,15 @@ WorkArea::save_meta_data()
 	{
 		String data;
 		GuideList::const_iterator iter;
-		for(iter=get_guide_list_x().begin();iter!=get_guide_list_x().end();++iter)
-		{
+		for (iter = get_guide_list().begin(); iter != get_guide_list().end(); ++iter){
 			if(!data.empty())
 				data+=' ';
-			data+=strprintf("%f",*iter);
+			data+=strprintf(" %f*%f*%f*%d", (*iter).point[0], (*iter).point[1], (*iter).angle.get(), (*iter).isVertical);
```

Comment:
I thought you said isVertical wasn't necessary anymore. I can't recall it correctly now :/

Reply:
No, it is still necessary for the current implementation. The reason being that horizontal and vertical rulers have a fundamental difference. the first having their center of rotation moving up and down at half the screen width, and the latter have their c.o.r moving left and right though half the screen height. This was to allow the rulers to reach more difficult places through GUI.

Reply:
isVertical removed :)